test/e2e/network_test.go

```diff
@@ -440,19 +440,22 @@ func (tc *testContext) getPodIP(selector metav1.LabelSelector) (string, error) {
 	return podList.Items[0].Status.PodIP, nil
 }
 
-// getWindowsServerContainerImage gets the appropriate WindowsServer image based on VXLAN port
+// getWindowsServerContainerImage gets the appropriate WindowsServer image
 func (tc *testContext) getWindowsServerContainerImage() string {
 	if tc.CloudProvider.GetType() == config.NonePlatformType {
 		// TODO: On platform=none we have to use 2004 since the job points to the WS2004 golden image via release repo
 		// When the release repo is updated to use 2022, this clause should be squashed with the above to use 2022
 		return "mcr.microsoft.com/powershell:lts-nanoserver-2004"
 	}
-	if tc.CloudProvider.GetType() == config.AWSPlatformType {
-		// On AWS we are currently testing 2019
+	// Check the Windows Server version environment variable to set the correct container image
+	windowsServerVersion := os.Getenv("WINDOWS_SERVER_VERSION")
```

Comment:
doing this check at the creation of `testContext`, and making `windowsServerVersion` part of the testContext struct will allow us to avoid having to make this syscall multiple times

Reply:
Addressed in https://github.com/openshift/windows-machine-config-operator/commit/c0b8dc12ef4652bca2551d4af0fa2d116b9b9984